javascript/create_retirement_graph_service.js

```diff
@@ -166,15 +180,23 @@ FinancialFreedom.service('CreateRetirementGraphService', ['DateService', functio
 
         drawIncomeLabel(chart, label_x, income_label_y);
     };
-    
+
     function addToolTipText(retirement_data) {
-        var date = retirement_data.intersection_point.x;
-        var expenses = retirement_data.intersection_point.y;
-        var asset_need = 25 * expenses * 12;
-        asset_need = Math.round(asset_need);
 
-        return "You will be able to safely <br>live off passive income in <br> <span class='bold'>" + date.getFullYear() + "</span>, when <br>you have total assets <br>of <span class='bold'>$" + numberWithCommas(asset_need) + "</span>.";
-        
+        if (retirement_data.intersection_point) {
+            var date = retirement_data.intersection_point.x;
+            var expenses = retirement_data.intersection_point.y;
+            var asset_need = 25 * expenses * 12;
+            asset_need = Math.round(asset_need);
+
+            return "You will be able to safely <br>live off passive income in <br> <span class='bold'>" + date.getFullYear() + "</span>, when <br>you have total assets <br>of <span class='bold'>$" + numberWithCommas(asset_need) + "</span>.";
```

Comment:
note, this usually doesn't apply to html (cause that can be more annoying than helpful), I mean just the js files